[Compile Error with 3.1.1]

I downloaded the latest openexr 3.1.1 from here.

Not sure why it fails but just in case someone may want to have a look, on linux
with latest GCC 11.2.0, I get:

    [ 42%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfPxr24Compressor.cpp.o
    [ 42%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfRational.cpp.o
    [ 42%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfRationalAttribute.cpp.o
    [ 43%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfRgbaFile.cpp.o
    [ 43%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfRgbaYca.cpp.o
    [ 43%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfRle.cpp.o
    [ 44%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfRleCompressor.cpp.o
    [ 44%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfScanLineInputFile.cpp.o
    [ 45%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfStandardAttributes.cpp.o
    [ 45%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfStdIO.cpp.o
    [ 45%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfStringAttribute.cpp.o
    [ 46%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfStringVectorAttribute.cpp.o
    [ 46%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfSystemSpecific.cpp.o
    [ 46%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfTestFile.cpp.o
    [ 47%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfThreading.cpp.o
    [ 47%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfTileDescriptionAttribute.cpp.o
    [ 47%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfTiledInputFile.cpp.o
    [ 48%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfTiledInputPart.cpp.o
    [ 48%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfTiledMisc.cpp.o
    [ 49%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfTiledOutputFile.cpp.o
    [ 49%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfTiledOutputPart.cpp.o
    [ 49%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfTiledRgbaFile.cpp.o
    [ 50%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfTileOffsets.cpp.o
    [ 50%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfTimeCode.cpp.o
    [ 50%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfTimeCodeAttribute.cpp.o
    [ 51%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfVecAttribute.cpp.o
    [ 51%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfVersion.cpp.o
    [ 51%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfWav.cpp.o
    [ 52%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfZip.cpp.o
    [ 52%] Building CXX object src/lib/OpenEXR/CMakeFiles/OpenEXR.dir/ImfZipCompressor.cpp.o
    [ 53%] Linking CXX shared library libOpenEXR-3_1.so
    [ 53%] Built target OpenEXR
    [ 53%] Building CXX object src/lib/OpenEXRUtil/CMakeFiles/OpenEXRUtil.dir/ImfCheckFile.cpp.o
    [ 54%] Building CXX object src/bin/exr2aces/CMakeFiles/exr2aces.dir/main.cpp.o
    [ 54%] Building CXX object src/bin/exrheader/CMakeFiles/exrheader.dir/main.cpp.o
    [ 55%] Building CXX object src/bin/exrmaketiled/CMakeFiles/exrmaketiled.dir/Image.cpp.o
    [ 55%] Linking CXX executable ../../../bin/exr2aces
    [ 55%] Built target exr2aces
    [ 55%] Building CXX object src/bin/exrstdattr/CMakeFiles/exrstdattr.dir/main.cpp.o
    [ 55%] Building CXX object src/bin/exrmaketiled/CMakeFiles/exrmaketiled.dir/main.cpp.o
    [ 55%] Linking CXX executable ../../../bin/exrheader
    [ 55%] Built target exrheader
    [ 55%] Building CXX object src/bin/exrmakepreview/CMakeFiles/exrmakepreview.dir/main.cpp.o
    [ 56%] Building CXX object src/bin/exrmakepreview/CMakeFiles/exrmakepreview.dir/makePreview.cpp.o
    [ 56%] Building CXX object src/bin/exrmaketiled/CMakeFiles/exrmaketiled.dir/makeTiled.cpp.o
    [ 57%] Linking CXX executable ../../../bin/exrstdattr
    [ 57%] Built target exrstdattr
    [ 57%] Building CXX object src/bin/exrenvmap/CMakeFiles/exrenvmap.dir/blurImage.cpp.o
    [ 57%] Linking CXX executable ../../../bin/exrmakepreview
    [ 57%] Built target exrmakepreview
    [ 57%] Building CXX object src/bin/exrenvmap/CMakeFiles/exrenvmap.dir/EnvmapImage.cpp.o
    [ 57%] Building CXX object src/lib/OpenEXRUtil/CMakeFiles/OpenEXRUtil.dir/ImfDeepImage.cpp.o
    [ 58%] Building CXX object src/bin/exrenvmap/CMakeFiles/exrenvmap.dir/main.cpp.o
    [ 59%] Building CXX object src/lib/OpenEXRUtil/CMakeFiles/OpenEXRUtil.dir/ImfDeepImageChannel.cpp.o
    [ 59%] Building CXX object src/bin/exrenvmap/CMakeFiles/exrenvmap.dir/makeCubeMap.cpp.o
    [ 60%] Building CXX object src/bin/exrenvmap/CMakeFiles/exrenvmap.dir/makeLatLongMap.cpp.o
    [ 61%] Linking CXX executable ../../../bin/exrmaketiled
    [ 61%] Built target exrmaketiled
    [ 62%] Building CXX object src/bin/exrmultiview/CMakeFiles/exrmultiview.dir/Image.cpp.o
    [ 62%] Building CXX object src/lib/OpenEXRUtil/CMakeFiles/OpenEXRUtil.dir/ImfDeepImageIO.cpp.o
    [ 62%] Building CXX object src/bin/exrenvmap/CMakeFiles/exrenvmap.dir/readInputImage.cpp.o
    [ 62%] Building CXX object src/bin/exrenvmap/CMakeFiles/exrenvmap.dir/resizeImage.cpp.o
    [ 62%] Building CXX object src/bin/exrmultiview/CMakeFiles/exrmultiview.dir/main.cpp.o
    [ 62%] Building CXX object src/lib/OpenEXRUtil/CMakeFiles/OpenEXRUtil.dir/ImfDeepImageLevel.cpp.o
    [ 63%] Building CXX object src/lib/OpenEXRUtil/CMakeFiles/OpenEXRUtil.dir/ImfFlatImage.cpp.o
    [ 64%] Linking CXX executable ../../../bin/exrenvmap
    [ 64%] Building CXX object src/bin/exrmultiview/CMakeFiles/exrmultiview.dir/makeMultiView.cpp.o
    [ 64%] Built target exrenvmap
    [ 64%] Building CXX object src/bin/exrmultipart/CMakeFiles/exrmultipart.dir/exrmultipart.cpp.o
    [ 64%] Building CXX object src/lib/OpenEXRUtil/CMakeFiles/OpenEXRUtil.dir/ImfFlatImageChannel.cpp.o
    [ 65%] Building CXX object src/lib/OpenEXRUtil/CMakeFiles/OpenEXRUtil.dir/ImfFlatImageIO.cpp.o
    [ 65%] Building CXX object src/lib/OpenEXRUtil/CMakeFiles/OpenEXRUtil.dir/ImfFlatImageLevel.cpp.o
    [ 66%] Linking CXX executable ../../../bin/exrmultiview
    [ 66%] Built target exrmultiview
    [ 67%] Building CXX object src/examples/CMakeFiles/OpenEXRExamples.dir/drawImage.cpp.o
    [ 67%] Building CXX object src/lib/OpenEXRUtil/CMakeFiles/OpenEXRUtil.dir/ImfImage.cpp.o
    [ 68%] Building CXX object src/lib/OpenEXRUtil/CMakeFiles/OpenEXRUtil.dir/ImfImageChannel.cpp.o
    [ 68%] Building CXX object src/examples/CMakeFiles/OpenEXRExamples.dir/generalInterfaceExamples.cpp.o
    [ 68%] Building CXX object src/lib/OpenEXRUtil/CMakeFiles/OpenEXRUtil.dir/ImfImageDataWindow.cpp.o
    [ 69%] Building CXX object src/examples/CMakeFiles/OpenEXRExamples.dir/generalInterfaceTiledExamples.cpp.o
    [ 69%] Building CXX object src/lib/OpenEXRUtil/CMakeFiles/OpenEXRUtil.dir/ImfImageIO.cpp.o
    [ 70%] Building CXX object src/lib/OpenEXRUtil/CMakeFiles/OpenEXRUtil.dir/ImfImageLevel.cpp.o
    [ 70%] Linking CXX executable ../../../bin/exrmultipart
    [ 70%] Built target exrmultipart
    [ 70%] Building CXX object src/test/OpenEXRCoreTest/CMakeFiles/CorePerfTest.dir/performance.cpp.o
    [ 70%] Building CXX object src/examples/CMakeFiles/OpenEXRExamples.dir/lowLevelIoExamples.cpp.o
    [ 70%] Building CXX object src/lib/OpenEXRUtil/CMakeFiles/OpenEXRUtil.dir/ImfSampleCountChannel.cpp.o
    [ 70%] Building CXX object src/examples/CMakeFiles/OpenEXRExamples.dir/main.cpp.o
    [ 71%] Building CXX object src/examples/CMakeFiles/OpenEXRExamples.dir/previewImageExamples.cpp.o
    [ 71%] Linking CXX shared library libOpenEXRUtil-3_1.so
    [ 71%] Building CXX object src/examples/CMakeFiles/OpenEXRExamples.dir/rgbaInterfaceExamples.cpp.o
    [ 71%] Built target OpenEXRUtil
    [ 71%] Building CXX object src/test/OpenEXRCoreTest/CMakeFiles/OpenEXRCoreTest.dir/base_units.cpp.o
    [ 71%] Linking CXX executable ../../../bin/CorePerfTest
    [ 71%] Built target CorePerfTest
    [ 71%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/compareB44.cpp.o
    [ 71%] Building CXX object src/test/OpenEXRCoreTest/CMakeFiles/OpenEXRCoreTest.dir/compression.cpp.o
    [ 71%] Building CXX object src/examples/CMakeFiles/OpenEXRExamples.dir/rgbaInterfaceTiledExamples.cpp.o
/home/Temp/rbt/openexr-3.1.1/src/test/OpenEXRCoreTest/compression.cpp: In function ‘bool withinB44ErrorBounds(const uint16_t (*)[4], const uint16_t (*)[4])’:
/home/Temp/rbt/openexr-3.1.1/src/test/OpenEXRCoreTest/compression.cpp:208:46: error: ‘imath_half_to_float’ was not declared in this scope
 208 | << std::dec << " (" << imath_half_to_float (Bbits)
 | ^~~~~~~~~~~~~~~~~~~
    [ 72%] Building CXX object src/test/OpenEXRCoreTest/CMakeFiles/OpenEXRCoreTest.dir/general_attr.cpp.o
/home/Temp/rbt/openexr-3.1.1/src/test/OpenEXRCoreTest/compression.cpp: In static member function ‘static void pixels::compareExact(uint16_t, uint16_t, int, int, const char*, const char*, const char*)’:
/home/Temp/rbt/openexr-3.1.1/src/test/OpenEXRCoreTest/compression.cpp:354:46: error: ‘imath_half_to_float’ was not declared in this scope
 354 | << std::dec << " (" << imath_half_to_float (a) << ") vs "
 | ^~~~~~~~~~~~~~~~~~~
make[2]: *** [src/test/OpenEXRCoreTest/CMakeFiles/OpenEXRCoreTest.dir/build.make:90: src/test/OpenEXRCoreTest/CMakeFiles/OpenEXRCoreTest.dir/compression.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
    [ 72%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/compareDwa.cpp.o
    [ 73%] Linking CXX executable OpenEXRExamples
    [ 73%] Built target OpenEXRExamples
    [ 74%] Building CXX object src/test/OpenEXRUtilTest/CMakeFiles/OpenEXRUtilTest.dir/main.cpp.o
    [ 75%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/compareFloat.cpp.o
    [ 75%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/main.cpp.o
    [ 76%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/random.cpp.o
    [ 76%] Building CXX object src/test/OpenEXRUtilTest/CMakeFiles/OpenEXRUtilTest.dir/testFlatImage.cpp.o
    [ 76%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testAttributes.cpp.o
    [ 76%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testB44ExpLogTable.cpp.o
    [ 77%] Building CXX object src/test/OpenEXRUtilTest/CMakeFiles/OpenEXRUtilTest.dir/testDeepImage.cpp.o
    [ 78%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testBackwardCompatibility.cpp.o
    [ 78%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testBadTypeAttributes.cpp.o
    make[1]: *** [CMakeFiles/Makefile2:1777: src/test/OpenEXRCoreTest/CMakeFiles/OpenEXRCoreTest.dir/all] Error 2
    make[1]: *** Waiting for unfinished jobs....
    [ 78%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testChannels.cpp.o
    [ 79%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testCompositeDeepScanLine.cpp.o
    [ 79%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testCompression.cpp.o
    [ 79%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testConversion.cpp.o
    [ 79%] Building CXX object src/test/OpenEXRUtilTest/CMakeFiles/OpenEXRUtilTest.dir/testIO.cpp.o
    [ 80%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testCopyDeepScanLine.cpp.o
    [ 80%] Linking CXX executable ../../../bin/OpenEXRUtilTest
    [ 80%] Built target OpenEXRUtilTest
    [ 80%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testCopyDeepTiled.cpp.o
    [ 81%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testCopyMultiPartFile.cpp.o
    [ 81%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testCopyPixels.cpp.o
    [ 81%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testCustomAttributes.cpp.o
    [ 82%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testDeepScanLineBasic.cpp.o
    [ 82%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testDeepScanLineHuge.cpp.o
    [ 82%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testDeepScanLineMultipleRead.cpp.o
    [ 83%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testDeepTiledBasic.cpp.o
    [ 83%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testDwaCompressorSimd.cpp.o
    [ 83%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testDwaLookups.cpp.o
    [ 84%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testExistingStreams.cpp.o
    [ 84%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testFutureProofing.cpp.o
    [ 85%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testHuf.cpp.o
    [ 85%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testIDManifest.cpp.o
    [ 85%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testInputPart.cpp.o
    [ 86%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testIsComplete.cpp.o
    [ 86%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testLargeDataWindowOffsets.cpp.o
    [ 86%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testLineOrder.cpp.o
    [ 87%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testLut.cpp.o
    [ 87%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testMagic.cpp.o
    [ 87%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testMalformedImages.cpp.o
    [ 88%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testMultiPartApi.cpp.o
    [ 88%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testMultiPartFileMixingBasic.cpp.o
    [ 89%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testMultiPartSharedAttributes.cpp.o
    [ 89%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testMultiPartThreading.cpp.o
    [ 89%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testMultiScanlinePartThreading.cpp.o
    [ 90%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testMultiTiledPartThreading.cpp.o
    [ 90%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testMultiView.cpp.o
    [ 90%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testNativeFormat.cpp.o
    [ 91%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testOptimized.cpp.o
    [ 91%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testOptimizedInterleavePatterns.cpp.o
    [ 91%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testPartHelper.cpp.o
    [ 92%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testPreviewImage.cpp.o
    [ 92%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testRgba.cpp.o
    [ 93%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testRgbaThreading.cpp.o
    [ 93%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testRle.cpp.o
    [ 93%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testSampleImages.cpp.o
    [ 94%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testScanLineApi.cpp.o
    [ 94%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testSharedFrameBuffer.cpp.o
    [ 94%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testStandardAttributes.cpp.o
    [ 95%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testTiledCompression.cpp.o
    [ 95%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testTiledCopyPixels.cpp.o
    [ 95%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testTiledLineOrder.cpp.o
    [ 96%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testTiledRgba.cpp.o
    [ 96%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testTiledYa.cpp.o
    [ 97%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testWav.cpp.o
    [ 97%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testXdr.cpp.o
    [ 97%] Building CXX object src/test/OpenEXRTest/CMakeFiles/OpenEXRTest.dir/testYca.cpp.o
    [ 98%] Linking CXX executable ../../../bin/OpenEXRTest
    [ 98%] Built target OpenEXRTest
    make: *** [Makefile:146: all] Error 2
    Now running `make -j4 install` from /home/Temp/rbt/openexr-3.1.1/BUILD_DIRECTORY/.
    Consolidate compiler generated dependencies of target Iex
    [ 2%] Built target Iex
    Consolidate compiler generated dependencies of target OpenEXRCore
    Consolidate compiler generated dependencies of target IexTest
    Consolidate compiler generated dependencies of target IlmThread
    [ 14%] Built target OpenEXRCore
    [ 15%] Built target IexTest
    [ 18%] Built target IlmThread
    Consolidate compiler generated dependencies of target exrinfo
    [ 19%] Built target exrinfo
    Consolidate compiler generated dependencies of target OpenEXR
    [ 53%] Built target OpenEXR
    Consolidate compiler generated dependencies of target exr2aces
    Consolidate compiler generated dependencies of target exrheader
Consolidate compiler generated dependencies of target exrmaketiled
    [ 54%] Built target exr2aces
    [ 54%] Built target exrheader
    [ 56%] Built target exrmaketiled
Consolidate compiler generated dependencies of target OpenEXRUtil
Consolidate compiler generated dependencies of target exrstdattr
Consolidate compiler generated dependencies of target exrmakepreview
    [ 57%] Built target exrstdattr
Consolidate compiler generated dependencies of target exrenvmap
    [ 58%] Built target exrmakepreview
Consolidate compiler generated dependencies of target exrmultiview
    [ 61%] Built target exrenvmap
    [ 66%] Built target OpenEXRUtil
Consolidate compiler generated dependencies of target exrmultipart
    [ 68%] Built target exrmultiview
    [ 68%] Built target exrmultipart
Consolidate compiler generated dependencies of target CorePerfTest
Consolidate compiler generated dependencies of target OpenEXRExamples
    [ 68%] Built target CorePerfTest
Consolidate compiler generated dependencies of target OpenEXRCoreTest
    [ 72%] Built target OpenEXRExamples
    [ 72%] Building CXX object src/test/OpenEXRCoreTest/CMakeFiles/OpenEXRCoreTest.dir/compression.cpp.o
Consolidate compiler generated dependencies of target OpenEXRUtilTest
    [ 72%] Building CXX object src/test/OpenEXRCoreTest/CMakeFiles/OpenEXRCoreTest.dir/main.cpp.o
    [ 74%] Built target OpenEXRUtilTest
    [ 74%] Building CXX object src/bin/exrcheck/CMakeFiles/exrcheck.dir/main.cpp.o
Consolidate compiler generated dependencies of target OpenEXRTest
    [ 97%] Built target OpenEXRTest
    [ 97%] Building CXX object src/test/OpenEXRCoreTest/CMakeFiles/OpenEXRCoreTest.dir/read.cpp.o
    [ 98%] Linking CXX executable ../../../bin/exrcheck
    [ 98%] Built target exrcheck
    [ 99%] Building CXX object src/test/OpenEXRCoreTest/CMakeFiles/OpenEXRCoreTest.dir/write.cpp.o
/home/Temp/rbt/openexr-3.1.1/src/test/OpenEXRCoreTest/compression.cpp: In function ‘bool withinB44ErrorBounds(const uint16_t (*)[4], const uint16_t (*)[4])’:
/home/Temp/rbt/openexr-3.1.1/src/test/OpenEXRCoreTest/compression.cpp:208:46: error: ‘imath_half_to_float’ was not declared in this scope
 208 | << std::dec << " (" << imath_half_to_float (Bbits)
 | ^~~~~~~~~~~~~~~~~~~
/home/Temp/rbt/openexr-3.1.1/src/test/OpenEXRCoreTest/compression.cpp: In static member function ‘static void pixels::compareExact(uint16_t, uint16_t, int, int, const char*, const char*, const char*)’:
/home/Temp/rbt/openexr-3.1.1/src/test/OpenEXRCoreTest/compression.cpp:354:46: error: ‘imath_half_to_float’ was not declared in this scope
 354 | << std::dec << " (" << imath_half_to_float (a) << ") vs "
 | ^~~~~~~~~~~~~~~~~~~

--- a/src/test/OpenEXRCoreTest/compression.cpp
+++ b/src/test/OpenEXRCoreTest/compression.cpp
@@ -49,6 +49,15 @@ using namespace IMATH_NAMESPACE;
 namespace IMF = OPENEXR_IMF_NAMESPACE;
 using namespace IMF;
 
+#if (IMATH_VERSION_MAJOR < 3) || (IMATH_VERSION_MAJOR == 3 && IMATH_VERSION_MINOR < 1)
+inline float imath_half_to_float( uint16_t a )
+{
+    half tmp;
+    tmp.setBits( a );
+    return static_cast<float>( tmp );
+}
+#endif
+
 ////////////////////////////////////////
 
 inline int

